File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/PlayerStoryboardSpecRenderer.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class PlayerStoryboardSpecRenderer{

	@SerializedName("spec")
	private String spec;

	public void setSpec(String spec){
		this.spec = spec;
	}

	public String getSpec(){
		return spec;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/PlayerStoryboardSpecRenderer.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class PlayerStoryboardSpecRenderer : Parcelable, Serializable {
    @SerializedName("spec")
    var spec: String? = null
    override fun toString(): String {
        return "PlayerStoryboardSpecRenderer{" +
                "spec = '" + spec + '\'' +
                "}"
    }
}